source/get-started/install-configfile.rst

```diff
@@ -15,67 +16,61 @@ machines.
 Prerequisites
 *************
 
-Ensure that your target system supports the installation:
+For installation onto bare metal, ensure that your target system 
+supports the installation:
 
 * :ref:`system-requirements`
 * :ref:`compatibility-check`
+* :ref:`bootable-usb` of the Live Server image
 
-Process
-*******
-
-This guide describes two methods for using a configuration file with the
-clr-installer tool. You can use either method to achieve the same goal. Choose
-the method that works best for your setup.
+Example 1: Fresh installation onto bare metal
+*********************************************
 
-If you are installing |CL| for the first time, we recommend Example 1.
+This example uses a YAML configuration file to perform a new installation.
 
-To clone an existing |CL| setup on another system, we recommend Example 2.
+#. Boot up the |CL| Live Server USB thumb drive.
 
-Example 1
-=========
+#. Select :guilabel:`Clear Linux OS` from the menu.
 
-This method uses a configuration file template to perform a new installation.
+#. In the console window, log in as `root` and set a password.
 
-Perform the following steps:
+#. Verify you have a network connection to the Internet and configure proxy
+   settings if you're working behind a firewall.
 
-#. Go to `Downloads`_ and download the latest Clear Linux OS Server image.
+#. Download a sample YAML configuration file. For example, if you want to 
+   install |CL| with a desktop GUI, you might want to use `live-desktop.yaml`_.
+   Or you can use the `live-server.yaml`_ if you want to install a non-GUI version
+   of |CL|.
 
-   For example:
-   https://download.clearlinux.org/releases/30010/clear/clear-30010-live-server.iso.xz
+   * Desktop:
 
-#. Follow the instructions to :ref:`bootable-usb` based on your OS.
+     .. code-block:: bash
 
-#. Boot up the USB thumb drive.
-#. Select :guilabel:`Clear Linux OS` from the menu.
-#. In the console window, log in as root and set a password.
-#. Verify you have a network connection to the Internet and configure proxy
-   settings if you're working behind a firewall.
-#. Download a :file:`live-server.yaml` template.
+        curl -O https://cdn.download.clearlinux.org/current/config/image/live-desktop.yaml
 
-   For example:
+    * Server:
 
-   .. code-block:: bash
+      .. code-block:: bash
 
-      curl -O https://download.clearlinux.org/releases/30010/clear/config/image/live-server.yaml
+         curl -O https://cdn.download.clearlinux.org/current/config/image/live-server.yaml
 
-#. Edit the template and change the settings as needed.
+#. Edit the YAML configuration file and change the settings as needed.

```

Comment:
Please give the YAML file a figure number. Then refer to that figure as "...shown below in Figure 1."

Reply:
It's a code-block.  Should we use a "Figure"?